modules/external_state/external-state.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://app.runx.dev/modules/external-state",
+  "type": "object",
+  "description": "Adds the ability to refer to external, preexisting terraform states in opta.",
+  "properties": {
+    "backend_type": {
+      "type": "string",
+      "description": "The type of backend this external state is using"
+    },
+    "config": {
+      "type": "string",
```

Comment:
we can use the object type like what we do with datadog or helm charts values
https://github.com/run-x/opta/blob/main/modules/datadog/datadog.json#L15-L18